src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay.tsx

```diff
@@ -225,10 +263,19 @@ const StatusDisplay = ({ titleStatus = "Polygon", mutate, refresh, name, record,
               type: "error",
               title: "Error!"
             });
+            setTimeout(() => {
+              setNotificationStatus({
+                open: false,
+                message: "",
+                type: "error",
+                title: "Error!"
+              });
+            }, 3000);
+            console.error(e);
           } finally {
-            refresh();
-            reloadEntity();
-            closeModal();
+            refresh && refresh();
+            reloadEntity && reloadEntity();
+            closeModal;
```

Comment:
I'm curious about this reversion. Isn't this a method call? Same question in similar block above.